src/auth/integration-tests/README.md

```diff
@@ -24,13 +24,13 @@ Credentials are up to date by running:
 gcloud auth application-default login
 ```
 
-Then, set the following environment variables and run the tests:
+Then, set the following environment variables and run the tests.
 
 ```sh
-env GOOGLE_CLOUD_PROJECT=rust-auth-testing-joonix
-    EXTERNAL_ACCOUNT_SERVICE_ACCOUNT_EMAIL=testsa@rust-auth-testing-joonix.iam.gserviceaccount.com
-    GOOGLE_WORKLOAD_IDENTITY_OIDC_AUDIENCE=//iam.googleapis.com/projects/246645052938/locations/global/workloadIdentityPools/google-idp/providers/google-idp
-  cargo test run_workload_ --features run-integration-tests --features run-byoid-integration-tests -p auth-integration-tests
+env GOOGLE_CLOUD_PROJECT=rust-external-account-joonix \
+EXTERNAL_ACCOUNT_SERVICE_ACCOUNT_EMAIL=testsa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com \
+GOOGLE_WORKLOAD_IDENTITY_OIDC_AUDIENCE=//iam.googleapis.com/projects/1092239828259/locations/global/workloadIdentityPools/google-idp/providers/google-idp \
+cargo test run_workload_ --features run-integration-tests --features run-byoid-integration-tests -p auth-integration-tests
```

Comment:
You may be able to avoid the magic number with:

```suggestion
GOOGLE_CLOUD_PROJECT=rust-external-account-joonix
GOOGLE_PROJECT_NUMBER=$(gcloud projects describe ${GOOGLE_CLOUD_PROJECT} --format='value(projectNumber)')

env GOOGLE_CLOUD_PROJECT=${GOOGLE_CLOUD_PROJECT} \
EXTERNAL_ACCOUNT_SERVICE_ACCOUNT_EMAIL=testsa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com \
GOOGLE_WORKLOAD_IDENTITY_OIDC_AUDIENCE=//iam.googleapis.com/projects/${GOOGLE_PROJECT_NUMBER}/locations/global/workloadIdentityPools/google-idp/providers/google-idp \
cargo test run_workload_ --features run-integration-tests --features run-byoid-integration-tests -p auth-integration-tests
```